version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.22-SNAPSHOT"
+version in ThisBuild := "0.0.23-SNAPSHOT"
```

Comment:
Should this be the "snapshot" version?

Reply:
Yeah, the release process releases the full version and then sets this file to the next snapshot version for development.